docs/reference/ccr/remote-recovery.asciidoc

```diff
@@ -0,0 +1,67 @@
+[role="xpack"]
+[testenv="platinum"]
+[[remote-recovery]]
+== Remote recovery
+
+Remote recovery is the process used to build a new copy of a shard on a follower
+node by copying data from the primary shard in the leader cluster. {es} uses this
+remote recovery process to bootstrap a follower index using the data from the
+leader index. This allows the follower to receive a copy of the current state of
+the leader index, even if a complete history of changes is not available on the
+leader due to Lucene segment merging.
+
+Remote recovery is a network intensive process that transfers all of the Lucene
+segment files from the leader cluster to the follower cluster. The follower
+requests that a recovery session be initiated on the primary shard in the leader
+cluster. The follower then requests file chunks concurrently from the leader. By
+default, the the process concurrently requests `5` large `1mb` file chunks as remote
+recovery is designed to support leader and follower clusters with high network
+latency between them.
+
+Information about an in-progress remote recovery can be obtained using the
+{ref}/cat-recovery.html[recovery API] on the follower cluster. Remote recoveries are implemented
+using the {ref}/modules-snapshots.html[snapshot and restore] infrastructure. This means that
+on-going remote recoveries will be labelled as type `snapshot` in the recovery api.
+
+The following setting can be used to rate-limit the data transmitted during remote
```

Comment:
We generally store these types of settings in the Elasticsearch Reference in pages like the ones linked here: https://www.elastic.co/guide/en/elasticsearch/reference/master/settings-xpack.html

I am happy to create that page if you agree.

Reply:
Yeah that makes sense.

Reply:
I've added those changes.  You can preview the new page in the Elasticsearch Reference here:
https://doc-review-63bf6.firebaseapp.com/es2/ccr-settings.html 

I've also updated the Remove recovery page in the Stack Overview to link to that content, which you can preview here:
https://doc-review-63bf6.firebaseapp.com/x4/remote-recovery.html